src/Pages/Sign/InputWithLabel.js

```diff
@@ -0,0 +1,28 @@
+import React, {Component} from "react"
+import "./InputWithLabel.scss"
+
+class InputWithLabel extends Component {
+  constructor(props){
+    super(props)
+  this.state={}
+}
```

Comment:
- 들여쓰기가 잘못 되어 있네요. eslint 설정 팀원들과 함께 확인해주세요. 다음과 같이 고쳐져야 합니다
- constructor에 인자로 props를 넘겨주는게 불필요한 상황 같습니다
![image](https://user-images.githubusercontent.com/35240142/90863664-6bd15000-e3ca-11ea-8423-441dbcaae74a.png)
```suggestion
class InputWithLabel extends Component {
  constructor() {
    super()
    this.state = {}
}
```
- 사실 현재 state를 사용하지 않고 있으므로 constructor가 불필요 합니다